core/block/editor/basic/extract_objects.go

```diff
@@ -35,29 +43,29 @@ func (bs *basic) ExtractBlocksToObjects(ctx session.Context, objectCreator Objec
 	rootIds := newState.SelectRoots(req.BlockIds)
 
 	for _, rootID := range rootIds {
-		rootBlock := newState.Pick(rootID)
-
-		objState := prepareTargetObjectState(newState, rootID, rootBlock, req)
+		var (
+			rootBlock        = newState.Pick(rootID)
+			objState         *state.State
```

Comment:
Mostly it is because of **err** declaration, that is shadowed multiple times in the function. So I tried to pre-declare other variables to reuse **err** already declared as return value